files/build_templates/init_cfg.json.j2

```diff
@@ -201,6 +201,9 @@
 {% endif %}
             "memory": "0M-2G:256M,2G-4G:320M,4G-8G:384M,8G-:448M",
             "num_dumps": "3"
+            "remote": "false",
```

Comment:
>remote [](http://example.com/codeflow?start=13&length=6)

Please do not change content in files/build_templates/init_cfg.json.j2.

Instead, please ensure the new feature work gracefully with old files/build_templates/init_cfg.json.j2. It could be disabled by default, but we want to keep default config unchanged.

The controller will deploy /etc/sonic/config_db.json, which will override files/build_templates/init_cfg.json.j2 content. The feature will be configured properly by /etc/sonic/config_db.json.